modules/skywalking-data-collect-protocol/9.7.0/patches/build_dot_bzl.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/common/BUILD b/common/BUILD
```

Comment:
Can you also upstream this patch to the original repo so that we won't need this in future version?